pkg/network/netlink/socket.go

```diff
@@ -162,6 +156,36 @@ func (s *Socket) ReceiveInto(b []byte) ([]netlink.Message, uint32, error) {
 	return msgs, netns, nil
 }
 
+// ParseNetlinkMessage parses b as an array of netlink messages and
```

Comment:
Not sure why this code block is here as this seems to be a copy of [`syscall.ParseNetlinkMessage`](https://cs.opensource.google/go/go/+/refs/tags/go1.18.2:src/syscall/netlink_linux.go;l=115) without any changes.